SETUP/CHANGELOG.md

```diff
@@ -3,20 +3,21 @@
 Major changes to this project are documented here. For minor changes,
 see the git history.
 
-## Deprecation notice
-The `project_pages` table and the `$writeBIGtable` variable in site_vars.php
-that controls writing to it are deprecated and will be removed in a future
-release. This table is not used anywhere within the code, but external code
-might rely on it.
-
 
 ## R??????
 Scripts supporting this upgrade are in `SETUP/upgrade/12`
 
 * Support Ubuntu 16.04 middleware (cpeel)
 * Support jpgraph version 4.2.0 (cpeel)
 * Move manager and sitemanager columns to usersettings (cpeel)
+* `$writeBIGtable` support, including `project_pages` table, has been
+  removed (cpeel)
```

Comment:
The 2 lines above belong to this PR, but not the 6 below.

Reply:
Yup, but this got you to look at them, didn't it :)

Reply:
Note that they are in a separate commit.